web.browser/src/components/PostToolbar/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes } from 'react';
+import { Toolbar, ToolbarGroup, ToolbarSeparator, ToolbarTitle } from 'material-ui/Toolbar';
+import MenuItem from 'material-ui/MenuItem';
+import styles from './styles.css'
+import FlatButton from 'material-ui/FlatButton';
+import {
+    Link,
+} from 'react-router';
+
+const PostToolbar = ({ sortNewest, sortPopular, orderBy }) => {
+    const flatbuttonCSS = {
```

Comment:
This function is called every time `PostToolbar` renders, so you're therefore re-assigning `flatButtonCss` on every render. Don't need to; move it outside this function.